l10n/en/firefox/all.ftl

```diff
@@ -4,31 +4,58 @@
 
 ### URL: https://www-dev.allizom.org/firefox/all/
 
-firefox-all-check-the-system-requirements = Check the system requirements
-firefox-all-release-notes = Release notes
-firefox-all-source-code = Source code
-firefox-all-need-help = Need help?
-firefox-all-which-browser-would = Which browser would you like to download?
+# HTML page title. Replace "English (US)" with your local language.
+firefox-all-download-the-firefox = Download the { -brand-name-firefox-browser } in English (US) and more than 90 other languages
```

Comment:
**nit:** should this be "Firefox browser"?

```suggestion
firefox-all-download-the-firefox = Download the { -brand-name-firefox } browser in English (US) and more than 90 other languages
```

Reply:
These show up as altered because I moved them up to the top of the file, they're not new strings.

That said, I might as well fix them while I'm in here.